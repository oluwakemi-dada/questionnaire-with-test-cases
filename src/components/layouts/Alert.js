import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  margin: 0 5% 1rem 5%;
`;

const AlertBg = styled.div`
  padding: 0.8rem 2rem;
  border-radius: 0.3rem;
  background: #f7dddc;
  color: #712b29;
  margin-bottom: 2rem;
  font-size: 1.5rem;
`;

const Alert = ({ msg }) => {
  return (
    <Wrapper>
      <AlertBg>{msg}</AlertBg>
    </Wrapper>
  );
};

export default Alert;
