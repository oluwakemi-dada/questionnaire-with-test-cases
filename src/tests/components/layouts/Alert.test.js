import React from 'react';
import { shallow } from 'enzyme';
import Alert from '../../../components/layouts/Alert';

test('should render Alert correctly with msg data', () => {
  const msg = 'Name is required';
  const wrapper = shallow(<Alert msg={msg} />);
  expect(wrapper).toMatchSnapshot();
});
