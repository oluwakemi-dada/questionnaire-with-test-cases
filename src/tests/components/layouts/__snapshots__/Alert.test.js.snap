// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Alert correctly with msg data 1`] = `
<styled.div>
  <styled.div>
    Name is required
  </styled.div>
</styled.div>
`;
